plugins/janus_sip.c

```diff
@@ -5785,10 +5804,6 @@ static void janus_sip_connect_sockets(janus_sip_session *session, struct sockadd
 	if(!session || (!audio_server_addr && !video_server_addr))
 		return;
 
-	if(session->media.updated) {
```

Comment:
This is removed because session->media.updated is always false on this place.

Reply:
Not sure why you think it will always be false there? Anyway, since it only showed an additional log line, no harm in getting rid of it, I guess.